logging_config.py

```
@@ -0,0 +1,30 @@
+                'level': logging.INFO,
+                'stream': 'ext://sys.stdout',
+            },
+        },
+        'root': {
+            'handlers': ['console'],
+            'level': logging.INFO,
+        },
+    })
+
+    logging.getLogger("py4j").setLevel(logging.ERROR)
+    logging.getLogger("py4j.java_gateway").setLevel(logging.ERROR)
```

Ruff

No newline at end of file

Add trailing newline

(W292)